docker/Makefile

```diff
@@ -71,12 +71,21 @@ build_% : repository
 		--tag pdo_$*:$(PDO_VERSION) \
 		--file '$(DOCKER_DIR)'/pdo_$*.dockerfile .
 
+build_sgx_services : repository
```

Comment:
Would be good to consistently name sgx targets; always prefix as elsewhere (or suffix) would be best ...

Reply:
Good point -- I have updated the code.